export.py

```diff
@@ -1798,21 +1813,24 @@ def get_instances_and_blocks(obs, rpass):
     scene = rpass.scene
     mb_on = scene.renderman.motion_blur
     mb_segs = scene.renderman.motion_segments
+    bake = rpass.bake
 
     for ob in obs:
         inst = get_instance(ob, rpass.scene, mb_on)
         if inst:
+            do_inst = False
             ob_mb_segs = ob.renderman.motion_segments if ob.renderman.motion_segments_override else mb_segs
 
             # add the instance to the motion segs list if transforming
-            if inst.transforming:
+            if inst.transforming and not bake:
```

Comment:
This might work, easier way to do it would above on line 1814 do

mb_on = False if bake else scene.renderman.motion_blur   just throw the global switch for motion blur.